tests/test_esm_runscripts/test_filedicts.py

```diff
@@ -0,0 +1,24 @@
+import yaml
+
+from esm_runscripts.filedicts import SimulationFile
+
+
+def test_filedicts_basics(fs):
```

Comment:
We can move the test cases as fixtures. I can do it later on.

Reply:
Not sure what you mean by this...?

Reply:
I mean the test cases and their expected results can be parameterized by a decorator.

Reply:
OK, that would be a separate PR though. For right now, I would keep the tests in a way that they can be written down as simply as possible, just make sure `test_blahblahblah` is in the name so `pytest` recognizes it.